subsys/mgmt/fmfu/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Full Modem Firmware Update Management(FMFU)"
+
+config MGMT_FMFU
+	bool "Serial Modem Firmware Update support"
+	depends on MCUMGR_SMP_UART
+	depends on MCUMGR
+	depends on NRF_MODEM_LIB
```

Comment:
Should this also have a !sys_init maybe

Reply:
I guess so if you want to ensure that you don't fail before main if the modem is bricked, I missed that in the review.